client-v2/src/util/configureStore.ts

```diff
@@ -15,25 +11,26 @@ import {
   persistClipboardOnEdit,
   persistOpenFrontsOnEdit
 } from './storeMiddleware';
+import { State } from 'types/State';
 
-export default function configureStore() {
+export default function configureStore(initialState: State) {
```

Comment:
in tests, this lets us inject fixture state into the store as it's used in the application, for end-to-end thunk testing. We've used mock-store previously, but I think I'm beginning to prefer this approach as it's as close to the in-app setup as possible. Interested in any thoughts 😄